### PR TITLE
Fix dimensions of mastodon icon

### PR DESCRIPTION
**Pull request by miltondp**
This PR changes the dimensions of the Mastodon icon so when the manuscript is converted to docx the image has the right size. @vincerubinetti might want to check this just in case.

This is how it looks right now and how it looks with this change:

image

image

### CODE REVIEW

**Comment by vincerubinetti** [comment]
Let's take this opportunity to make the icon consistent with the others. Here is cleaner code from Font Awesome:

```svg
<!-- Modified from https://fontawesome.com/icons/mastodon under its CC BY 4.0 License -->
<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 448 512">
	<path fill="#808080" d="M433 179.11c0-97.2-63.71-125.7-63.71-125.7-62.52-28.7-228.56-28.4-290.48 0 0 0-63.72 28.5-63.72 125.7 0 115.7-6.6 259.4 105.63 289.1 40.51 10.7 75.32 13 103.33 11.4 50.81-2.8 79.32-18.1 79.32-18.1l-1.7-36.9s-36.31 11.4-77.12 10.1c-40.41-1.4-83-4.4-89.63-54a102.54 102.54 0 0 1-.9-13.9c85.63 20.9 158.65 9.1 178.75 6.7 56.12-6.7 105-41.3 111.23-72.9 9.8-49.8 9-121.5 9-121.5zm-75.12 125.2h-46.63v-114.2c0-49.7-64-51.6-64 6.9v62.5h-46.33V197c0-58.5-64-56.6-64-6.9v114.2H90.19c0-122.1-5.2-147.9 18.41-175 25.9-28.9 79.82-30.8 103.83 6.1l11.6 19.5 11.6-19.5c24.11-37.1 78.12-34.8 103.83-6.1 23.71 27.3 18.4 53 18.4 175z" />
</svg>
```

Also preserving aspect ratio doesn't really matter in this case, as the svg will scale to be contained within the shorter dimension without stretching. That is, keeping it 16x16 will just add a tiny bit of white space on either side, in this case.

In the future, or maybe we can do it in this PR, it'd be better to only specify the height, and to specify it in terms of the text size it is within, i.e. `height="1em"`.

**Comment by agitter** [comment]
Thanks for working on this @miltondp.

> it'd be better to only specify the height, and to specify it in terms of the text size it is within

Can we address that in this same pull request? @vincerubinetti is the proposal to edit the height in the svg files and leave the relevant parts of `content/00.front-matter.md` as

`ORCID icon{.inline_icon width=16 height=16}`

for all icons?

**Comment by vincerubinetti** [comment]
Apologies, I remember what is going on now.

We already have an `.inline_icon` class that sets the height to `1em`, but because that CSS (in `themes/default.html`) is loaded at the end of the `<body>`, it won't get applied until a few hundred milliseconds after the content of `<body>` has already been shown. This leads to a momentary flash where the icons are huge because they have no set dimensions. So I probably had us add `width="16" height="16"` to the icons themselves as a fallback, just for that momentary flash. `1em` (or any other font size) will override it once the style loads, and preserve aspect ratio.

tl;dr, just add `width="16" height="16"` to each svg icon, and only attach `{:.inline_icon}` when using it. No more is needed.

Also for consistency, check Font Awesome first for any icons you add in the future (I know, FA probably didn't have Mastodon right away). You'll get a nice clean single `<path>` element and matching `viewBox`, and that's all you need.

**Comment by miltondp** [comment]
I hope I got @vincerubinetti right and made the correct changes.

**Comment by vincerubinetti** [comment]
The width and height next to .inline_icon is not necessary.

File: content/00.front-matter.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Published: {{manubot.date_long}}
   {%- endif %}
   {%- if author.mastodon is defined and author.mastodon is not none and author["mastodon-server"] is defined and author["mastodon-server"] is not none %}
     {%- set has_ids = true %}
-    · ![Mastodon icon](images/mastodon.svg){.inline_icon}
+    · ![Mastodon icon](images/mastodon.svg){.inline_icon width=16 height=16}
```
[the remark this answers is not in the document]
Comment (vincerubinetti) [reply content/00.front-matter.md:55]: > like for the docx export?

That very well may be. I always forget about docx. If you could do a quick local test (my machine was recently wiped and I can't quickly run Manubot atm)... remove the dimensions next to `.inline_icon` and see if the docx displays properly in Word. If it's smart enough, it could set the dimensions from the inherent SVG dimensions (attached directly in the SVGs).

Regardless though, the html/pdf exports should only need one or the other. If docx needs dimensions next to `.inline_icon`, we can get rid of them in the SVGs themselves.

[the remark this answers is not in the document]
Comment (miltondp) [reply content/00.front-matter.md:55]: Yes, I confirm that if I remove the dimensions next to `.inline_icon`, then all icons look big and blurry in the docx file (see below), but the HTML/PDF exports work as expected.

image

If I leave the dimensions next to `.inline_icon` but remove them from the SVG, then everything works in all exports and the icons look even better in docx (see below, before/after):

output

[the remark this answers is not in the document]
Comment (vincerubinetti) [reply content/00.front-matter.md:55]: Thanks for checking and doing this PR Milton. Can you [enable editing of your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests) so I can make some final tweaks?

[the remark this answers is not in the document]
Comment (miltondp) [reply content/00.front-matter.md:55]: @vincerubinetti thank you, just pushed those changes. We should be ready to merge.

**Review by agitter — APPROVED** [review]
Will let @dhimmel take another look and then merge.
